homeassistant/components/cover/strings.json

```diff
@@ -1,10 +1,31 @@
 {
-    "device_automation": {
-        "condition_type": {
-            "is_open": "{entity_name} is open",
-            "is_closed": "{entity_name} is closed",
-            "is_opening": "{entity_name} is opening",
-            "is_closing": "{entity_name} is closing"
-        }
+  "device_automation": {
+    "condition_type": {
+      "is_open": "{entity_name} is open",
+      "is_closed": "{entity_name} is closed",
+      "is_opening": "{entity_name} is opening",
+      "is_closing": "{entity_name} is closing",
+      "is_position": "Current {entity_name} position",
+      "is_tilt_position": "Current {entity_name} tilt position"
+    },
+    "trigger_type": {
+      "is_open": "{entity_name} opened",
+      "is_closed": "{entity_name} closed",
+      "is_opening": "{entity_name} opening",
+      "is_closing": "{entity_name} closing",
+      "is_position": "{entity_name} position",
+      "is_tilt_position": "{entity_name} tilt position"
+    },
+    "action_type": {
+      "open": "Open {entity_name}",
+      "close": "Close {entity_name}"
+      "toggle": "Toggle {entity_name}"
```

Comment:
Toggle seems like a weird description. You don't really toggle a cover ?

Reply:
Right, but it's supported by covers. What could be a better name?